src/commands.rs

```diff
@@ -318,6 +344,25 @@ impl AssetCommand {
         Ok(Assets::new(serde_json::from_str(&response.text()?)?))
     }
 
+    pub fn set_headers(
+        &self,
+        id: &str,
+        headers: Vec<(String, String)>,
+    ) -> anyhow::Result<(), CommandError> {
+        let endpoint = format!("v4/assets?id=eq.{}", id);
+        let map: HashMap<_, _> = headers.into_iter().collect();
+        patch(&self.authentication, &endpoint, &json!({ "headers": map }))
+    }
+
+    pub fn inject_js(&self, id: &str, js_code: &str) -> anyhow::Result<(), CommandError> {
```

Comment:
I guess it is designed to be separate commands (set_headers and inject_js)- right?

Reply:
yes